apps/site-landing/src/containers/HelpDocs/FaqMenu.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { Link } from 'react-router-dom';
+import styles from "./styles.module.less";
+import { List, Rail } from "semantic-ui-react";
+import { GreaterThanMobileSegment, MobileSegment } from "@thecointech/shared/components/ResponsiveTool";
+
+type Props = {
+    categories: any
+  }
+  
+function displayEntry(index: number, name: string ){
+    return <List.Item key={index}><Link to={"/faq/theme-"+((name)?.split("-"))[0].replace(/ /g, '')}>{((name.split("-"))[1])}</Link>
```

Comment:
the `split` happening here seems to be a very common theme.

I'm not sure what it is supposed to do.  From my POV, it's a very opaque mechanism for selecting things.  It might pay to create a smaller helper function just to make clear whats happening:

```tsx
<List.Item key={index}>
  <Link to={"/faq/theme-"+getWhatsit(name)')}>
     {getThingy(name)}
  </Link>
</>
```
This has the advantage of moving all the complexity of -how- your selecting your wotsit & thingy away somewhere else, so you don't have to reason about that at the same time as your reasoning about the <list>